src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -446,7 +473,9 @@ bool CHyprGroupBarDecoration::onMouseButtonOnDeco(const Vector2D& pos, wlr_point
         return true;
 
     // click on padding
-    if (BARRELATIVEX - (m_fBarWidth + BAR_HORIZONTAL_PADDING) * WINDOWINDEX > m_fBarWidth) {
+    const auto TABPAD   = !*PSTACKED && (BARRELATIVEX - (m_fBarWidth + BAR_HORIZONTAL_PADDING) * WINDOWINDEX > m_fBarWidth);
+    const auto STACKPAD = *PSTACKED && (BARRELATIVEY - (m_fBarHeight + BAR_PADDING_OUTER_VERT) * WINDOWINDEX < BAR_PADDING_OUTER_VERT);
```

Comment:
shouldn't this be `> m_fBarHeight`?

Reply:
No, BARRELATIVEY turns out to start at the beginning of the vertical padding (0 -> padding[start of bar]->end) so detecting if you're in the padding is different. There's slight differences in stacked/tabbed padding layouts due to the extra vertical padding in stacked orientation.

Reply:
I see, my bad